### ISSUE TITLE
Windows thinks taisei is malware.

### ISSUE THREAD

**Issue by angrypig555** [taisei-project/taisei#379]
So i just downloaded it and windows smartscreen doesn't want me to run it. Is it safe?

**Comment by Akaricchi** [comment]
You can check it with multiple antivirus engines here: https://www.virustotal.com/

I've checked both the installer and the main game executable (taisei.exe). [The installer](https://www.virustotal.com/gui/file/6fe132fe510850b16ea21efe4555d6429366a7357981a0dff9b49a9641b1b7bd) alerted only 1 engine out of 72. [The main exe](https://www.virustotal.com/gui/file/7b3797a3b410c05b4ff7220cf2d269e080af0cf37203518f724ca505855dc470) didn't trigger anything at all. So it looks like NSIS, the installer we're using, triggers a false positive.

In other words, no, Taisei is not malware, this is just Windows being paranoid. If you don't believe me, you're free to inspect the source code and build it yourself.
